src/com/aca/Client.java

```diff
@@ -0,0 +1,83 @@
+package com.aca;
+
+import com.aca.exceptions.WrongProductData;
+import com.aca.items.*;
+import com.aca.types.Gadget;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class Client {
+
+    public static void main(String[] args) throws IOException {
+        List<Gadget> gadgets = getProductList("aca-java-june-homeworks/src/com/aca/productList/product.csv");
+        Manager manager = new Manager(gadgets);
+
+       manager.testGadgets(gadgets);
+        System.out.println(manager.getWorkingGadgets());
+       System.out.println(manager.getNotWorkingGadgets());
+        for (Gadget gadget : gadgets) {
+            System.out.println(gadget.getName());
+        }
+        manager.sellItem("Watch");
+
+    }
+    private static List<Gadget> getProductList(String listPath){
```

Comment:
let's move this logic into some DataConverter class, which we will provide to our client also